### PR TITLE
Test jac_structure!

### PR DESCRIPTION
**Pull request by abelsiqueira**

### CODE REVIEW

File: src/qn_model.jl [hunk 1]
```diff
@@ -57,6 +57,7 @@ for meth in (:jprod!, :jtprod!)
   @eval $meth(nlp :: QuasiNewtonModel, x :: AbstractVector, y :: AbstractVector, z :: AbstractVector) = $meth(nlp.model, x, y, z)
 end
 jac_structure(nlp :: QuasiNewtonModel) = jac_structure(nlp.model)
```
Comment (dpo) [inline src/qn_model.jl]: Maybe `jac_structure()` is no longer necessary?!

**Comment by dpo** [comment]
The breakage job fails with `ERROR: LoadError: KeyError: key "GITHUB_AUTH" not found`.

**Comment by abelsiqueira** [comment]
Rebased and working now. This PR is from outside JSO, so there is no breakage test. But everything is already broken from the last PR, so it shouldn't make a difference.

**Comment by dpo** [comment]
Obrigado.

**Pull request merged**